### ISSUE TITLE
Save and Load images scripts Windows OS detection gets 2009 as version on Windows 20H2

### ISSUE THREAD

**Issue by izaac** [rancher/rancher#32698]
**What kind of request is this:**
bug

**Steps to reproduce (least amount of steps as possible):**

- use the save or load images scripts for populating a private registry

On v2.5.8

https://github.com/rancher/rancher/releases/download/v2.5.8/rancher-save-images.ps1
https://github.com/rancher/rancher/releases/download/v2.5.8/rancher-loadimages.ps1

**Result:**

- the OS detection detects Windows as 2009 on Windows 20H2

```
$os_release_id = $(Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\' | Select-Object -ExpandProperty ReleaseId)

powershell -command "Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\' | Select-Object -ExpandProperty ReleaseId"
2009
```

This build the suffix as `<image-name>-windows-2009` instead of `<image-name>-windows-20H2`

Results in `manifest unknown` errors while pulling.

**Comment by sowmyav27** [comment]
Can validate this issue along with this - https://github.com/rancher/rancher/issues/33403

**Comment by deniseschannon** [comment]
Available to test in v2.6.0-rc1

**Comment by izaac** [comment]
Rancher version: `v2.6.0-rc2`

Validated the `rancher-save-images.ps1` and the `rancher-load-images.ps1` scripts are now properly auto detecting the 20H2 SAC version using it as suffix.
